### PR TITLE
feedback on your-first-cluster docs via macos

### PR DESCRIPTION
**Pull request by jackfrancis**
A bunch of suggestions after going through the macos flow!

### CODE REVIEW

File: your-first-cluster.md [hunk 1]
```diff
@@ -289,36 +288,28 @@ spec:
...
+Time to create the Cluster! It's as simple as using the yaml spec we've prepared::
 
 ```bash
 kubectl apply -f yamls/clusters/docker-cluster-one.yaml
 ```
 
-Now that the Cluster has been created we can watch it come into being with:
+Now that the Cluster has been created we can use `clusterctl` to describe its components:
 ```bash
-watch clusterctl describe cluster docker-cluster-one
```
Comment (jackfrancis) [inline your-first-cluster.md:303]: Macos doesn't ship w/ `watch` by default, and I don't think we want to worry about recommending folks due `brew install watch` or some other thing, so I've simply reformulated these instructions a bit to not watch progress. Below I updated the `watch kubectl...` command with a `kubectl .... -w` flavor, should have the same effect.

File: your-first-cluster.md [hunk 2]
```diff
@@ -116,10 +116,9 @@ spec:
           replicas: 1                    # The number of worker nodes in this MachineDeployment
 ```
 
-The Cluster definition above defines some of the fundamental characteristics of our Cluster - it's name, the version of Kubernetes to install, the number of nodes in the control plane, the number and type of worker nodes, as well as some details about networking.
+The Cluster specification above defines some of the fundamental characteristics of our Cluster - its name, the version of Kubernetes to install, the number of nodes in the control plane, the number and type of worker nodes, as well as some details about networking.
 
-
-Time to create the Cluster! It's as simple as:
+Time to create the Cluster! It's as simple as using the yaml spec we've prepared::
```
Comment (sbueringer) [inline your-first-cluster.md]:
```suggestion
Time to create the Cluster! It's as simple as using the yaml spec we've prepared:
```

File: your-first-cluster.md (hunk 1 above)
Comment (sbueringer) [reply your-first-cluster.md:303]: Goot catch!

**Review by killianmuldoon — COMMENTED** [review]
Thanks for the revision!

Looks good to me

File: your-first-cluster.md [hunk 3]
```diff
@@ -464,10 +455,9 @@ spec:
...
```
Comment (sbueringer) [inline your-first-cluster.md]:
```suggestion
Time to create the Cluster! It's as simple as using the yaml spec we've prepared:
```

File: your-first-cluster.md [hunk 4]
```diff
@@ -289,36 +288,28 @@ spec:
           replicas: 1                    # The number of worker nodes in this MachineDeployment
 ```
 
-The Cluster definition above defines some of the fundamental characteristics of our Cluster - it's name, the version of Kubernetes to install, the number of nodes in the control plane, the number and type of worker nodes, as well as some details about networking.
-
+The Cluster specification above defines some of the fundamental characteristics of our Cluster - its name, the version of Kubernetes to install, the number of nodes in the control plane, the number and type of worker nodes, as well as some details about networking.
 
-Time to create the Cluster! It's as simple as:
+Time to create the Cluster! It's as simple as using the yaml spec we've prepared::
```
Comment (sbueringer) [inline your-first-cluster.md]:
```suggestion
Time to create the Cluster! It's as simple as using the yaml spec we've prepared:
```

**Review by sbueringer — COMMENTED** [review]
Thx good improvements!!

**Comment by sbueringer** [comment]
Merged my suggestions (`::` => `:`)

Merging the PR now

**Pull request merged**